cmd/gen-go-sample/README.md

```diff
@@ -0,0 +1,62 @@
+Sample Generator for Go API Clients
+===========================
+
+A generator for samples showing how to make API calls with clients generated by [gapic-generator-go](https://github.com/googleapis/gapic-generator-go).
+
+Prerequisites
+===========================
+- proto descriptors of an API
+- sample configuration written in gapic config*
+
+Note: We are in the process of moving sample configuration out of gapic config into a separate sample config yaml file.
+Refer to sample config spec for how to [config a sample](https://github.com/googleapis/gapic-generator/blob/master/src/main/proto/com/google/api/codegen/samplegen/v1p2/sample_config_v1p2.proto). Before that, samples can be configured using [gapic config](https://github.com/googleapis/gapic-generator/blob/master/src/main/proto/com/google/api/codegen/config.proto).
```

Comment:
Can you change both of these links to be based on a commit hash rather than `master`?

From the GitHub UI, viewing a file, press `y` to change the link to the current commit being viewed